systers_portal/templates/meetup/snippets/users-grid.html

```diff
@@ -0,0 +1,16 @@
+<div class="row mt40">
```

Comment:
Can you remove the margin top? It adds too much whitespace and looks like a gap that missing something.